core/arch/aarch64/codec.h

```diff
@@ -30,8 +30,8 @@
  * DAMAGE.
  */
 
-#ifndef CODEC_H
-#define CODEC_H 1
+#ifndef CODEC_H_
```

Comment:
Please update the code style wiki page to add this convention of leading `_` and trailing `_H_`